src/test/super-admins.test.js

```diff
@@ -1,3 +1,154 @@
-test('', () => {
-  expect(2).toBe(2);
+import request from 'supertest';
+import app from '../app';
+import superAdmin from '../models/Super-admins';
+import superAdminSeeds from '../seed/super-admins';
+
+describe('Super-admins - Unit tests', () => {
+  beforeAll(async () => {
+    await superAdmin.collection.insertMany(superAdminSeeds);
+  });
+
+  const mockedSuperAdmin = {
+    name: 'Pablo',
+    last_name: 'Schivazappa',
+    email: 'pablo.schivazappa@gmail.com',
+    password: 'blablabla',
+  };
+
+  // eslint-disable-next-line no-underscore-dangle
+  const validId = superAdminSeeds[0]._id;
+  const invalidID = '6349bc420340abc705a7e8a5naid';
+  const IdNotFound = '6349bc420340abc705a72222';
+
+  describe('GET /Super-admins', () => {
+    test('should return status code 200', async () => {
+      const response = await request(app).get('/super-admins').send();
+      expect(response.status).toBe(200);
```

Comment:
```suggestion
      expect(response.status).toBe(200);
      expect(response.body.error).toBeFalsy();
      expect(response.body.data.length).toBeGreaterThan(0);
```

Making two new separated tests for those two expects is a bit unnecessary and both of them can be inserted in the first test

Reply:
I want to see more comments from you guys!! Thanks @DamianPalavecino 👏